config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+
+  post "/builds", :controller => "submissions", :action => "submit"
```

Comment:
Not sure what this Hound comment is about

> Prefer single-quoted strings when you don't need string interpolation or special symbols.

Since the style guide I thought it uses says to prefer double-quoted strings always (possible exception: within an interpolation expression).

Ignore this comment for now and I'll look into what's up with the style guide.